convert.go

```diff
@@ -50,7 +50,7 @@ func Translate(c *Config) error {
 	defer bfd.Flush()
 
 	// Write the header. This makes e.g. Github ignore diffs in generated files.
-	if _, err = fmt.Fprintf(bfd, "// Code generated by go-bindata. (@generated) DO NOT EDIT.\n\n //Package %s generated by go-bindata.", c.Package); err != nil {
+	if _, err = fmt.Fprintf(bfd, "// Code generated by go-bindata. (@generated) DO NOT EDIT.\n\n//Package %s generated by go-bindata.", c.Package); err != nil {
```

Comment:
If you're gonna persist with this, a better approach is going to be to call format.Source on the generated byte output, since the output of that is guaranteed to be correct.

Alternatively, you'll want to run `go fmt ./...` and check whether it generates a Git diff. See github.com/kevinburke/differ for a tool that can help enforce this rule.